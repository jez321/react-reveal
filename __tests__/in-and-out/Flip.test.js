/*
 * Flip Component Test Suite
 *
 * Copyright Â© Roman Nosov 2017
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE.txt file in the root directory of this source tree.
 */
import Flip from '../../Flip';

import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import { configure, shallow } from 'enzyme';
import React from 'react';

configure({ adapter: new Adapter() });


describe('Flip', () => {
  it('renders a initial view', () => {
    const content = shallow(
      <Flip bottom right>
        <div>Test test</div>
      </Flip>
    );
    expect(content.html()).toMatchSnapshot();
  });
});
