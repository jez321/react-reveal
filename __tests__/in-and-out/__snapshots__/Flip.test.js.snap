// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Flip renders a initial view 1`] = `"<div class="react-reveal">Test test</div>"`;
